### ISSUE TITLE
unable to install

### ISSUE THREAD

**Issue by zozo6015** [ahupowerdns/metronome#15]
when I run make I get the following error message:

 #make
g++ -Wall -O3 -ggdb -I. -I yahttp/yahttp  -pthread -MMD -MP -DYAHTTP_MAX_URL_LENGTH=16384 -std=gnu++0x    -c -o metronome.o metronome.cc
In file included from metronome.cc:1:0:
yahttp/yahttp/yahttp.hpp:12:27: fatal error: yahttp-config.h: No such file or directory
 #include "yahttp-config.h"
                           ^
compilation terminated.
make: *** [metronome.o] Error 1

**Comment by wbdv** [comment]
Create an empty file:
````bash
touch yahttp/yahttp/yahttp-config.h
````

After that, I got other errors:
````bash
/usr/bin/ld: cannot find -lboost_program_options
/usr/bin/ld: cannot find -lstdc++
````

and for this, you need to edit sysdeps/Linux.inc and add -L/usr/lib64/ on the third line (I have x64)

STATICFLAGS ?=-Wl,**-L/usr/lib64/** -Bstatic [..]

Anyway, I manage to compile and run, but I cannot retrive any data :(

**Comment by zozo6015** [comment]
my errors are on yahttp side.

**Comment by zozo6015** [comment]
I just needed to install doxygen then everything went fine.

**Comment by zozo6015** [comment]
you only need to install libboost-program-options1.49-dev and libstdc++ then you will be able to compile it with no issues. I have mine running however when I try to browse the web address I get 404 file not found

**Comment by ahupowerdns** [comment]
Can you join our IRC channel to discuss? 

"IRC (instant messaging)
We hang out in #powerdns and #powerdns-dev on OFTC (irc.oftc.net)."

	Bert

**Comment by zozo6015** [comment]
sure let me just install a client.

**Comment by zozo6015** [comment]
After setting up a web server (I personally use nginx) and copy the files from html folder into the location where the web server is running from I had the carbon-ourname set with a dot "." which apparently is a bug and will be fixed on the next release of powerdns. After I changed the name from some.name to some-name and I removed all the files from the stats folder which was related to some.name everything worked just fine. Thanks for the help.
